apps/builder/app/builder/features/style-panel/property-label.tsx

```diff
@@ -133,7 +137,22 @@ export const PropertyInfo = ({
 
   return (
     <Flex direction="column" gap="2" css={{ maxWidth: theme.spacing[28] }}>
-      <Text variant="titles">{title}</Text>
+      <Flex justify="between">
+        <Text variant="titles" truncate>
+          {title}
+        </Text>
+        {link && (
+          <Link
+            href={link}
+            target="_blank"
+            rel="noreferrer"
+            color="inherit"
+            variant="inherit"
+          >
+            <ExternalLinkIcon size={13} />
+          </Link>
```

Comment:
Should this be moved to design system. So next time it can just be picked like `IconButton` ?

Reply:
yeah, I agree, we need an IconLink component in the design system

Reply:
and use it everywhere, easy to find currently the places by searching for ExternalLinkIcon usages